modules/040-node-manager/images/bashible-apiserver/pkg/template/context_builder.go

```diff
@@ -145,15 +145,16 @@ func (cb *ContextBuilder) Build() (BashibleContextData, map[string][]byte, map[s
 		versionMapWrapper: versionMapFromMap(cb.versionMap),
 		RunType:           "Normal",
 		Normal: normal{
-			BootstrapTokenPath: "/var/lib/bashible/bootstrap-token",
 			ClusterDomain:      cb.clusterInputData.ClusterDomain,
 			ClusterDNSAddress:  cb.clusterInputData.ClusterDNSAddress,
+			BootstrapTokens:    cb.clusterInputData.BootstrapTokens,
 			ApiserverEndpoints: cb.clusterInputData.APIServerEndpoints,
 			KubernetesCA:       cb.clusterInputData.KubernetesCA,
 		},
-		Registry: cb.registryData,
-		Images:   cb.imagesDigests,
-		Proxy:    cb.clusterInputData.Proxy,
+		Registry:      cb.registryData,
+		Images:        cb.imagesDigests,
+		Proxy:         cb.clusterInputData.Proxy,
+		PackagesProxy: cb.clusterInputData.Proxy,
```

Comment:
I think we need use another field for PackagesProxy

Reply:
https://github.com/deckhouse/deckhouse/pull/7751/files#diff-4cf1142188bcbd5aa64109bc19449f191d6576893f4a0362a6123c42bfbba123R221